.idea/other.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`.idea/` files should not be committed and should be added to `.gitignore` instead.

Reply:
Will update in upcoming PR